src/pageEditor/fields/MultiSelectWidget.tsx

```diff
@@ -45,6 +45,7 @@ const MultiSelectWidget: React.FC<MultiSelectWidgetProps> = ({
       isDisabled={disabled}
       isClearable={isClearable}
       options={options}
+      {...(props.id ? { inputId: props.id } : {})}
```

Comment:
This fixes an actual bug - the label was not properly connected to this control component because this was missing the inputId.

Reply:
What's wrong with just passing it directly? If id is undefined, then it's the same as not passing it at all isn't it? If it can be null, we can null coalesce it to undefined

```suggestion
     inputId={props.id}
```